.github/actions/release-please-semvar/action.yml

```diff
@@ -32,3 +32,40 @@ runs:
         git push origin v${{ steps.release.outputs.major }}
         git push origin v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }}
       if: ${{ steps.release.outputs.release_created }}
+    - name: tag component major and minor versions
+      if: ${{ steps.release.outputs.release_created }}
+      uses: actions/github-script@v7
+      env:
+        release_output: ${{ toJSON(steps.release.outputs) }}
+      with:
+        result-encoding: string
+        script: |
+          exec.exec('git config user.name czi-github-helper[bot]')
+          exec.exec('git config user.email 95879977+czi-github-helper[bot]@users.noreply.github.com')
+          
+          const outputs = JSON.parse(process.env.release_output);
+          // for some reason pathsReleased is a string and not an array within the JSON
+          const pathsReleased = JSON.parse(outputs.paths_released);
+          pathsReleased.forEach((release_path) => {
+            if (release_path === ".") {
+                // ignore root package
+                return;
+            }
+            const major = outputs[`${release_path}--major`];
+            const minor = outputs[`${release_path}--minor`];
+            const package_name = outputs[`${release_path}--name`].split(":")[0];
+            
+            console.log(`Creating ${major}.${minor} releases for ${package_name}`);
+
+            const cmdOpts = {
+              ignoreReturnCode: true
+            };
+            exec.exec('git', ["tag", "-d", `${package_name}-v${major}`], cmdOpts);
+            exec.exec('git', ["tag", "-d", `${package_name}-v${major}.${minor}`], cmdOpts);
+            exec.exec('git', ["push", "origin", `:${package_name}-v${major}`], cmdOpts)
+            exec.exec('git', ["push", "origin", `:${package_name}-v${major}.${minor}`], cmdOpts)
+            exec.exec('git', ["tag", "-a", `${package_name}-v${major}`, "-m", `Release v${major}`])
+            exec.exec('git', ["tag", "-a", `${package_name}-v${major}.${minor}`, "-m", `Release v${major}.${minor}`])
+            exec.exec('git', ["push", "origin", `${package_name}-v${major}`])
+            exec.exec('git', ["push", "origin", `${package_name}-v${major}.${minor}`])
```

Comment:
I think you want to `await` all these `exec` calls (and refactor to use standard `for` loop instead of `forEach` in support of using `await`)

Reply:
done and tested

```
 * [new tag]         fake_package_name-v12 -> fake_package_name-v12
/usr/bin/git push origin fake_package_name-v12.3
To https://github.com/chanzuckerberg/argus-example-app
 * [new tag]         fake_package_name-v12.3 -> fake_package_name-v12.3
 ```